assets/src/design-system/components/banner/index.js

```diff
@@ -105,9 +106,10 @@ export const Banner = forwardRef(
         {...rest}
       >
         <Title
+          as="h1"
```

Comment:
I'm not sure this should be heading level 1, since that should really be the overall heading for the entire view. Thoughts on h2? I'm kinda thinking about it like an outline: 

- h1: DASHBOARD / EDITOR 
--> h2: Telemetry banner (has the context of existing in the dashboard or editor)
---> Context about telemetry 
--> h2: Nav 
---> List items and info 
--> h2: canvas 
--> h2: stories

Reply:
yeah that makes way more sense. h2 it shall be!